### DISCUSSION TITLE
BACKPACK_LICENSE  is not working

### DISCUSSION THREAD

**Discussion by mahendrasahu421** [Laravel-Backpack/community-forum#1205] [Q&A (Help)]
i have BACKPACK_LICENSE how to use this i add this on .env file or config/backpack/crud.php but its not working

**Comment by karandatwani92** [comment]
Hello @mahendrasahu421

You don't need to configure the key in the project. To know how to use it:
1. Visit https://backpackforlaravel.com/user/tokens
2. Go to the **Instructions** tab
